.code-samples.meilisearch.yaml

```diff
@@ -560,3 +559,22 @@ geosearch_guide_sort_usage_2: |-
   curl -X POST 'http://localhost:7700/indexes/restaurants/search' \
     -H 'Content-type:application/json' \
     --data-binary '{ "sort": ["_geoPoint(48.8583701,2.2922926):asc", "rating:desc"] }'
+document_guide_create_index_primary_key: |-
```

Comment:
I think, but please feel free to ignore, that we should keep a consistency between the code-samples name at all times. Thus, removing it from one code sample creates a first inconsistency. Removing them from all at the same time will keep that consistency. 

On another note the reason why `document_guide` is added is that it gives the information of where it is used. Meaning that: 
- It ensures that it will not be used in two different places and thus we ensure that when removed, we do not accidentally forget about changing the sample-id in the other places. As one-sample = one use. 
- It gives the information for the integration team for example to know where to look in the documentation to understand the context of the code sample that we are changing.

Reply:
@bidoubiwa I see, I didn't realize that there was an existing naming pattern like this for code samples. It has been inconsistently followed in the past so sticking to it here might not have a lot of value, but why not keep `document_guide` until we move the code sample (or [update our naming standards](https://github.com/meilisearch/documentation/issues/1320)). If you truly don't mind changing it later, I accept and will resolve this conversation 😄